### DISCUSSION TITLE
鹰眼图VcOverviewMap能否实现与主视图的联动？

### DISCUSSION THREAD

**Discussion by kfcvs** [zouyaoji/vue-cesium#205] [Q&A]
例如点击鹰眼图中的某点，在主视图中也进行显示。
此外，鹰眼图能否自行控制放大缩小？

**Comment by zouyaoji** [comment]
我本地以及改了个版本，和你说的这个需求有点像，不过还没更新，下个版本你测测哈。

**Reply by zouyaoji** [reply]
@kfcvs 看看这个呢

https://zouyaoji.top/vue-cesium/#/zh-CN/component/controls/vc-overview-map

**Reply by kfcvs** [reply]
效果很棒！是我想要的，但能不能提一个需求：
添加鹰眼图默认打开或关闭的属性；
麻烦了！

**Reply by zouyaoji** [reply]
将在v3.0.13版本支持。
